checkstyle-config/src/main/resources/suppressions.xml

```diff
@@ -20,6 +20,7 @@ limitations under the License.
 <suppressions>
   <!-- For our tests we do not demand API documentation. -->
   <suppress files=".*\.java" checks="Javadoc.*"/>
+  <suppress files=".*\.java" checks="CustomImportOrder" />
```

Comment:
Does this also allow any order for the imports as well as the newline?

Reply:
https://checkstyle.sourceforge.io/config_imports.html#CustomImportOrder